modules/Header/ui/CategoriesNav/CategoriesNavMobile.tsx

```diff
@@ -0,0 +1,114 @@
+import type { TranslatedCategory } from '@prezly/sdk';
+import type { Locale } from '@prezly/theme-kit-nextjs';
+import { translations } from '@prezly/theme-kit-nextjs/index';
+
+import { FormattedMessage } from '@/adapters/client';
+import { app } from '@/adapters/server';
+import { Dropdown } from '@/components/Dropdown';
+
+import { CategoryButton } from './CategoryButton';
+import { CategoryItem } from './CategoryItem';
+import { FeaturedCategory } from './FeaturedCategory';
+
+import styles from './CategoriesNavMobile.module.scss';
+
+export async function CategoriesNavMobile({
+    categories,
+    localeCode,
+    navigationItemClassName,
+    navigationItemButtonClassName,
+}: CategoriesNavMobile.Props) {
+    const showAllCategories = categories.length < 4;
+
+    const categoriesList = await app().categories();
+
+    function getCategory(translatedCategory: TranslatedCategory) {
+        return categoriesList.find((i) => i.id === translatedCategory.id)!;
+    }
+
+    function isCategoryFeatured(translatedCategory: TranslatedCategory) {
+        return getCategory(translatedCategory).is_featured;
+    }
+
+    const featuredCategories = categories.filter(isCategoryFeatured);
+    const regularCategories = categories.filter((i) => !isCategoryFeatured(i));
+
+    if (showAllCategories) {
```

Comment:
I'm not sure how to feel about this 🤔 Here it looks like the "Media galleries" is a title and the items below are media galleries, but those are actually categories.

![localhost_3000_(iPhone 12 Pro)](https://github.com/prezly/theme-nextjs-bea/assets/4209081/cb9b197f-292d-4cde-abaa-f37766915163)

![localhost_3000_(iPhone 12 Pro) (1)](https://github.com/prezly/theme-nextjs-bea/assets/4209081/323b8dd0-21d9-45df-bbd4-fb3ddeb22839)

If I don't have any featured categories, the categories themselves look completely different from the other elements? 🤔 

![localhost_3000_(iPhone 12 Pro) (2)](https://github.com/prezly/theme-nextjs-bea/assets/4209081/07f196d4-82a2-4c23-8d58-dbefd5334b0f)

Reply:
a14afe38